### ISSUE TITLE
Are we intending to keep Veterans Employment Center in the footer?

### ISSUE THREAD

**Issue by emilyville** [department-of-veterans-affairs/vets-website#2218]
please discuss! @koconnor719 @gnakm @ayaleloehr 

I thought we'd planned on removing it with launch of Careers and Employment. It might be worth having just the "tools" - Job Search, resume builder, etc called out here instead of the trademarked name

**Comment by koconnor719** [comment]
was this something you guys negotiated w/ rosye? (to keep VEC in footer?) @gnakm

**Comment by gnakm** [comment]
We did tell Rosye that we will be keeping VEC in the footer. 

However now that all of our templates are changing and usability sessions are showing us that Veterans are looking for specific tools (ex. military skills translator, job search), I think we can make a case for putting only tools in the footer. I think it would bother our stakeholders if we changed the link in the footer from VEC to Careers and Employment but this approach is something that is consistent with everything else we have on the site. 

If we put tools in the footer, are we putting _all_ tools in the footer or just popular tools? If it's just most used tools, maybe we should wait until Q4 to make those changes.
